inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/sink/EventStat.java

```diff
@@ -17,20 +17,29 @@
 
 package org.apache.inlong.dataproxy.sink;
 
+import java.util.concurrent.atomic.AtomicInteger;
 import org.apache.flume.Event;
+import org.apache.inlong.dataproxy.base.OrderEvent;
+import org.apache.inlong.dataproxy.utils.MessageUtils;
 
 public class EventStat {
+
+    private static long RETRY_INTERVAL_MS = 1 * 1000L;
     private Event event;
-    private int myRetryCnt;
+    private AtomicInteger myRetryCnt = new AtomicInteger(0);
```

Comment:
It is better to retry count use CountDownLatch.

Reply:
The retry mechanism will be redesigned later